helpers/helpers.v1.d/go

```diff
@@ -130,36 +120,51 @@ ynh_install_go() {
 
     # Install or update goenv
     mkdir -p $goenv_install_dir
-    pushd "$goenv_install_dir"
-    if ! [ -x "$goenv_install_dir/bin/goenv" ]; then
-        ynh_print_info --message="Downloading goenv..."
-        git init -q
-        git remote add origin https://github.com/syndbg/goenv.git
+    goenv="$(command -v goenv $goenv_install_dir/bin/goenv | grep "$goenv_install_dir/bin/goenv" | head -1)"
```

Comment:
This command will fail if goenv is not present. You should something like that instead:
```
if command -v …
```

Reply:
In that case

https://github.com/YunoHost/yunohost/blob/a96cb9bb2c1a74806b9571daf4009652d46b35bb/helpers/helpers.v1.d/ruby#L135
and
https://github.com/YunoHost/yunohost/blob/a96cb9bb2c1a74806b9571daf4009652d46b35bb/helpers/helpers.v2.1.d/ruby#L77

should not be ok

Reply:
To me what's troublesome is : what is it doing in the first place x_X

It looks like this is meant to verify that `$RBENV_INSTALL_DIR/bin/rbenv` exists (and maybe is executable) ? Shouldn't we just use something like 

```
rbenv="$RBENV_INSTALL_DIR/bin/rbenv"
if [[ -x "$rbenv" ]]
then
     ...
else
     ...
fi
```

(In fact that was the original code x_x)